README.md

```diff
@@ -35,6 +35,15 @@ Alfresco Content Services deployed via `docker-compose` or Kubernetes contains t
 * [Deploying using Docker Compose](docs/docker-compose-deployment.md)
 * [Customizing your deployment](docs/customising-deployment.md)
 
+## Production environments
+The HELm charts in this repository provide a PostgreSQL database in a docker container and do not configure
```

Comment:
A small typo here "HELm"

Reply:
We agreed, Alex, that small typos would just be done directly rather than holding up the review.